sdk/search/Microsoft.Azure.Search/tests/Tests/SkillsetsTests.cs

```diff
@@ -1113,7 +1113,8 @@ private static Skillset CreateTestSkillsetWebApiSkill(bool includeHeader = true)
                     description: "A simple web api skill",
                     context: RootPathString)
             {
-                HttpMethod = "POST"
+                HttpMethod = "POST",
+                DegreeOfParallelism = 7
```

Comment:
Where's the test that looks for this 7 in a deserialized Web API skill?

Reply:
This is asserted in `CreateAndValidateSkillset` where we check that the response of the create request (response body of a 201) matches what we passed in as a request